logs/latest.log

```diff
@@ -0,0 +1,76 @@
+[13:11:13] [main/INFO] [LaunchWrapper]: Loading tweak class name net.minecraftforge.fml.common.launcher.FMLTweaker
```

Comment:
We need to remove the logs.

Can you do a git rm -rf logs and then add logs/ to .gitignore?